mantidproject/mantid
# Add import of simpleapi to scripts generated by script builder

This PR adds the import of `mantid.simpleapi` to scripts generated by the algorithm history interface

**To test:**

1. Load a worksapce and perform varius algorithms (rebin, crop, ect.)
2. right click on workspace and open History
3. click on copy to clipboard
4. open a new script in the editor and clear the default inports.
5. paste the copied script and check that the fist line is `from mantid.simplaeapi import *`

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
